docs/_source/getting-started/installation-guide.rst

```diff
@@ -104,11 +104,10 @@ For users interested in the latest features and developments, you can install PO
     Refer back to the recommended installation steps, starting from :ref:`point 4 <posydon-env>`, to download the required dataset and set the necessary environment variables.
 
 
-Using POSYDON on HPC Facilities
+Running grids using POSYDON on HPC Facilities
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-(TODO: check whether it is still needed)
-If you are planning to run POSYDON's population synthesis on a High-Performance Computing (HPC) facility, it's essential to have `mpi4py` installed to enable parallel computations. 
+If you are planning to create MESA grids using POSYDON, it's essential to have `mpi4py` installed to enable parallel computations.
```

Comment:
I know this section is for POSYDON on HPC facilities, but I think here we could clearly state (especially for very novice users) that mpi4py is essential for HPC usage. It is less so for personal machines, is that right?
```suggestion
If you are planning to create MESA grids using POSYDON on HPC facilities, it's essential to have `mpi4py` installed to take advantage of parallel computations.
```

Reply:
This is because MESA requires it, right? 

For population runs, you don't need it anymore.